config.yaml

```diff
@@ -105,6 +105,7 @@ teams:
   - Dindexx
   - JoTiTu
   - ntsbs
+  - kenkosmowski
```

Comment:
You need to add this line above:
`  members:`

Being a maintainer of a team is distinct from being a maintainer of a project.